app/config/ConfigKeys.scala

```diff
@@ -43,6 +43,7 @@ object ConfigKeys {
   val mtdSignUpFeature: String = "features.mtdSignUp.enabled"
   val ddCollectionInProgressFeature: String = "features.ddCollectionInProgress.enabled"
   val displayCovidMessageFeature: String = "features.displayCovid.enabled"
+  val mergedMTDfBStatuses: String = "features.mergedMTDfBStatuses.enabled"
```

Comment:
There is no mention of a feature switch in the story, and following the discussion in our team channel I don't think one is necessary.